python/ray/autoscaler/updater.py

```diff
@@ -223,18 +229,19 @@ def do_update(self):
         # Run init commands
         self.provider.set_node_tags(self.node_id,
                                     {TAG_RAY_NODE_STATUS: "setting-up"})
-
         m = "{}: Initialization commands completed".format(self.node_id)
         with LogTimer("NodeUpdater: {}".format(m)):
             with open("/dev/null", "w") as redirect:
                 for cmd in self.initialization_commands:
-                    self.ssh_cmd(cmd, redirect=redirect)
+                    self.ssh_cmd(
+                        cmd, redirect=None if self.verbose else redirect)
```

Comment:
Can we make it verbose by default? It's really bad if we hang without messages, and only a minor nuisance to be verbose.

Reply:
Done